testsuite/integration-arquillian/servers/app-server/jboss/wildfly/pom.xml

```diff
@@ -30,33 +30,90 @@
     <name>App Server - Wildfly</name>
     
     <properties>
-        <wildfly.server.version>23.0.2.Final</wildfly.server.version>
+        <wildfly.server.version>28.0.1.Final</wildfly.server.version>
         <app.server.jboss>wildfly</app.server.jboss>
         
         <app.server.jboss.groupId>org.wildfly</app.server.jboss.groupId>
         <app.server.jboss.artifactId>wildfly-dist</app.server.jboss.artifactId>
         <app.server.jboss.version>${wildfly.server.version}</app.server.jboss.version>
         <app.server.jboss.unpacked.folder.name>wildfly-${wildfly.server.version}</app.server.jboss.unpacked.folder.name>
-        
-        <app.server.saml.adapter.artifactId>keycloak-saml-wildfly-adapter-dist</app.server.saml.adapter.artifactId>
-
-        <skip.elytron.adapter.installation>false</skip.elytron.adapter.installation>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.keycloak.testsuite</groupId>
             <artifactId>integration-arquillian-servers-app-server-spi</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jboss.arquillian.protocol</groupId>
+                    <artifactId>arquillian-protocol-servlet</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.arquillian.protocol</groupId>
+            <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
```

Comment:
Looks like no version is provided for this artifact here, or in any of the parent POM files. Is this dependency carried as a transitive dependency and the version determined there? If so, is that considered best practice?

Asking this for my own reference, since I am not as familiar with Maven and it's best practices.

Reply:
I think we are using the arquillian bom for this [here](https://github.com/keycloak/keycloak/blob/25.0.0/testsuite/integration-arquillian/pom.xml#L175-L182). We are doing the same with quarkus so I suppose it's OK.